### ISSUE TITLE
Hidden text in role=listbox is read while using IE11

### ISSUE THREAD

**Issue by blm126** [nvaccess/nvda#7672]
### Description
Hidden text inside of a role=option in a listbox is read by NVDA in IE only.

### Steps to reproduce:
1. Open https://jsfiddle.net/14opp52t/3/ in IE11
2. Focus on the listbox. 

### Expected behavior:
Elements with display:none should not be read by NVDA.

### Actual behavior:
The elements are read.

### System configuration:
NVDA version: 2017.3
NVDA Installed or portable: portable
Windows version: Windows 10 Pro Version 1703 15063.674

Name and version of other software in use when reproducing the issue: Internet Explorer 11

### Other questions:
Does the issue still occur after restarting your PC? Yes
Have you tried any other versions of NVDA? No

**Comment by Adriani90** [comment]
@blm126 I cannot reproduce this issue in IE11 with NVDA 2019.1.1. Do you still see this issue?

**Comment by blm126** [comment]
No, I can no longer reproduce this issue.
